### DISCUSSION TITLE
Clicking

### DISCUSSION THREAD

**Discussion by animafps** [aiming-taxonomy/glossary#10] [Concepts]
Current definition:
`The skill to move your mouse to a target to click/ shoot with a semi-auto weapon.`

**Comment by deLiGHT-03** [comment]
The ability to quickly and accurately click a target at the moment your crosshair and target intersect.

**Reply by animafps** [reply]
Thats click timing not the type of aim clicking

**Reply by deLiGHT-03** [reply]
I don't consider "clicking" to be a type of aiming. That seems a bit muddled to me.

Clicking is just click-timing. Sometimes leveraged into position by mouse control, sometimes just holding angles and waiting for targets to pass through the crosshair. But they're both click-timing.

**Reply by animafps** [reply]
-_- 
We are trying to define the concept that is in the category of types of aim to help define specific training scenarios, situations and techniques in general. everyone knows that clicking is just exactly that clicking the mouse but we are defining a more abstract idea of again the categories of situations.
click-timing is a skill which falls under the clicking type of aim and is a movement that is used when aiming in a clicking situation/ scenario.
While i dont want to be rude but thats not what we are trying to define in this discussion thread feel free to suggest a better name for this but i want to stay on the same idea and define and name it

**Reply by deLiGHT-03** [reply]
Oh I see, you want to broaden it so you can include strategies of approach. That makes perfect sense then.

**Comment by riddbtw** [comment]
Here's the definition from Kovaak:

> Player uses a semi-automatic weapon to fire at a target that either they manually aimed at with their mouse or the target moved in front of their crosshair and the player had to click at the correct time to do damage.

I like that, it's pretty succinct and captures both use cases (the 'flick' requirement, or waiting for the target to pass through the crosshair).

Should probably specify "aka Click-timing"
